Services.Test/TestConstants.Properties.dbl

```diff
@@ -35,9 +35,13 @@ namespace Services.Test
                         begin
                             try
                             begin
-                                instance = JsonSerializer.Deserialize<TestConstants>(File.ReadAllText(Path.Combine(Environment.GetEnvironmentVariable("SOLUTIONDIR"), "Services.Test", "TestConstants.Values.json")))
+                                data filePath = Path.Combine(Environment.GetEnvironmentVariable("SOLUTIONDIR"), "Services.Test", "TestConstants.Values.json")
+                                if (File.Exists(filePath)) then
+                                    instance = JsonSerializer.Deserialize<TestConstants>(File.ReadAllText(filePath))
+                                else
+                                    instance = new TestConstants()
                             end
-                            catch (e, @Exception)
+                            catch (e, @JsonException)
```

Comment:
I dont think we should be silently failing here, the user should really know about invalid json

Reply:
Fixed. See 57749c739dd187b6a069b54de936a4b2e8f07117